modules/google-maps/src/utils.js

```diff
@@ -0,0 +1,140 @@
+/* global document, google */
+import {Deck} from '@deck.gl/core';
+
+export function createDeckInstance(map, overlay, deck) {
+  if (deck) {
+    if (deck.props.userData.map === map) {
```

Comment:
Since this is called `userData` and open to the user, and this is a geospatial library where `map` is a common concept, would we want to be a bit more careful to avoid name conflict? e.g. `deck.props.userData._googleMap`?